mopidy_musicbox_webclient/static/js/controls.js

```diff
@@ -604,6 +613,36 @@ function showFavourites() {
     });    
 }
 
+function upgradeStreamUrisToFavourites() {
+    $.cookie.json = true;
```

Comment:
Is this function something that we should flag for removal with a TODO type statement? I assume once everyone has transitioned to >= v2.1.0 this won't be needed anymore?

Maybe in release 3.x or 4.x?